### ISSUE TITLE
SSL broken on Arch Linux

### ISSUE THREAD

**Issue by acidicX** [synergy/synergy#4645]
The SSL feature only works when started as root. On a normal account, it just seems to try to download the "ns" plugin and silently fails to do that.
On the next restart, it complains about the plugin missing:
````
2015-05-09T17:19:49 ERROR: failed to load plugin: libns.so
	/build/synergy/src/synergy-1.7.2-stable/src/lib/arch/unix/ArchPluginUnix.cpp,79
2015-05-09T17:19:50 FATAL: An error occurred: libssl.so.10: cannot open shared object file: No such file or directory
	/build/synergy/src/synergy-1.7.2-stable/src/lib/synergy/App.cpp,127
NOTE: stopping synergy desktop process
````

I guess it tries to save the plugin somewhere without having proper permissions?
This is on Arch x64
More users are having this issue:
https://bbs.archlinux.org/viewtopic.php?id=197150

**Comment by acidicX** [comment]
Also, there is a bug entry in the arch linux bugtracker with additional info:
https://bugs.archlinux.org/task/44768

**Comment by speaker** [comment]
This issue should be fixed by #4696, however, only to the extent that we can try to provide the necessary parts during install. If an OS installer reports that a dependency is met but it isn't, (64bit libssl, and 32bit libcrypto, for example) there may still be problems unless additional changes are met on the OS side.

**Comment by acidicX** [comment]
Seems to work on arch now, at least for me. Thanks!
